Software/Python/grovepi.py

```diff
@@ -329,6 +330,26 @@ def dht(pin, module_type):
 	else:
 		return [float('nan'),float('nan')]
 
+# after a list of numerical values is provided
+# the function returns a list with the outlier(or extreme) values removed
+# make the std_factor_threshold bigger so that filtering becomes less strict
+# and make the std_factor_threshold smaller to get the opposite
+def statisticalNoiseReduction(values, std_factor_threshold = 2):
```

Comment:
While the number of data manipulation functions we have is rather low, I think we shouldn't come up with new a library, since it would induce confusion for the users. 
For the moment, I think keeping it inside the `grovepi.py` library is fine.

We will definitely hold onto that on the future implementations - it's something we've also thought about.